### PR TITLE
docs: Added link to clean air calculation pdf

### PR DESCRIPTION
**Pull request by angrox**
Added link to the theory pdf.

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ The documentation covers all information around the UV-C clean air units.
 
 ## DIY Hardware
 
+### Calculations
+
+The effiency calculation can be found [here](theory.pdf).
```
Comment (thorsten) [inline documentation/README.md:9]: May be we should add more here, e.g. 

`The effiency calculation of the device can be found [in this PDF document](theory.pdf).`

**Review by thorsten — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ Here are some reading information sources, if you want to read up on various opi
 - https://www.cuimc.columbia.edu/news/far-uvc-light-safely-kills-airborne-coronaviruses
 - https://pubmed.ncbi.nlm.nih.gov/32547908/
 
+We also have a [calculation (PDF)](documentation/theory.pdf) of the theoretical effiency of the clean air device.
```
Comment (mlkiefer) [inline README.md:33]:
```suggestion
We estimated the irradiance inside the device as described in the [calculation (PDF)](documentation/theory.pdf).
```

File: documentation/README.md (hunk 1 above)
Comment (mlkiefer) [inline documentation/README.md:9]:
```suggestion
Estimations about the irradiance and considerations about the room size for which such a device could be feasible are located [here](theory.pdf).
```

**Review by mlkiefer — APPROVED** [review]

**Review by mlkiefer — DISMISSED** [review]
I've added some suggestions

**Review by thegreatdane6 — APPROVED** [review]

**Pull request merged**